cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller.go

```diff
@@ -452,7 +452,9 @@ func (c *machineController) scalableResourceProviderIDs(scalableResource *unstru
 			}
 		}
 
-		klog.Warningf("Machine %q has no providerID", machine.GetName())
+		if klog.V(3).Enabled() {
```

Comment:
could we make this log level 4 to keep it consistent with the other debug log statement in the capi provider?